src/elements/content-sidebar/Sidebar.js

```diff
@@ -9,11 +9,16 @@ import DetailsSidebar from './DetailsSidebar';
 import SkillsSidebar from './SkillsSidebar';
 import ActivitySidebar from './ActivitySidebar';
 import MetadataSidebar from './MetadataSidebar';
+import { mark } from '../../utils/performance';
 import {
     SIDEBAR_VIEW_SKILLS,
     SIDEBAR_VIEW_ACTIVITY,
     SIDEBAR_VIEW_DETAILS,
     SIDEBAR_VIEW_METADATA,
+    ORIGIN_DETAILS_SIDEBAR,
```

Comment:
Should these follow the `SIDEBAR_ORIGIN_SKILLS` convention used above for views?